spec/features/favourites_feature_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+feature 'favouriting photos' do
+
+  scenario 'a user can favourite a photo, which updates the photo favourite count', js: true do
```

Comment:
Metrics/LineLength: Line is too long. [95/90] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)